### DISCUSSION TITLE
Does delayed tasks follow concrrency and rate limiter?

### DISCUSSION THREAD

**Discussion by laneme** [taskforcesh/bullmq#2291] [Q&A] [answered]
I have task that delays itself (with `DelayedError`) until it is complete, it can delay itself upto 10 times or so.

I have a `concurrency: 1` and also rate limited to `{ duration: 1000, max: 1 }`.

I'm curious if multiple tasks are delayed into same point of time, are concurrency/limiter is still respected.

**Comment by roggervalf** [comment, marked as answer]
hi @laneme, yes it should respect concurrency/limiter logic

**Reply by laneme** [reply]
aha thank you for confirming :)
